lib/version-util.js

```diff
@@ -179,6 +179,16 @@ function findGreatestByLevel(versions, current, level) {
         .value();
 }
 
+/**
+ * @param versions  Array of all available versions
+ * @Returns         An array of versions with the release versions filtered out
+ */
```

Comment:
Done. But now after moving it to npm.js, I'm not sure how to write tests for it since it will local and not exported via anything.